react-ui/src/components/Banks.js

```diff
@@ -37,8 +37,13 @@ const Banks = (props) => {
 
   useEffect(() => {
     const fetchBanks = async () => {
-      const result = await axios.get("/user/getBanks");
-      setBanks(result.data.banks);
+      await axios
+        .get("/user/getBanks")
+        .then((response) => {
+          setBanks(response.data.banks);
+        })
+        .catch((err) => console.log({ err }));
+

```

Comment:
We don't need the `.then` since it would defeat the purpose of async and await.
We could do a try catch block instead.
I'll check if it works as intended.

```
try {
    const result = await axios.get("user/getBanks")
    setBanks(result.data.banks);
}
catch(e){
     console.log(e);
}
```